packages/file_selector/file_selector_platform_interface/LICENSE

```diff
@@ -0,0 +1,25 @@
+Copyright 2020 The Flutter Authors. All rights reserved.
```

Comment:
(I'm not sure all other licenses are from 2020?)

Reply:
See recent discussion in the ecosystem channel; there isn't yet a standardized date for this repo.

Reply:
Is this resolved?